docs/08-guides.md

```diff
@@ -189,6 +189,44 @@ import { repeat } from "/web_modules/lit-html/directives/repeat.js";
 // ✔ snowpack installed: lit-html, lit-element [0.25s]
 ```
 
+### Svelvet
```

Comment:
Lets just call this section Svelte, I feel like this is what most users will be looking for.